cranelift/codegen/src/isa/riscv64/abi.rs

```diff
@@ -635,11 +640,12 @@ impl ABIMachineSpec for Riscv64MachineDeps {
         fixed_frame_storage_size: u32,
         outgoing_args_size: u32,
     ) -> FrameLayout {
-        let mut regs: Vec<Writable<RealReg>> = regs
-            .iter()
-            .cloned()
-            .filter(|r| DEFAULT_CALLEE_SAVES.contains(r.to_reg().into()))
-            .collect();
+        let is_callee_saved = |reg: &Writable<RealReg>| match call_conv {
+            isa::CallConv::Patchable => true,
```

Comment:
Similar question to aarch64 (or feel free to resolve with no comment if you respond to aarch64)